js-packages/@prestojs/util/src/pagination/PageNumberPaginator.ts

```diff
@@ -50,6 +53,12 @@ export default class PageNumberPaginator extends Paginator<
         if (typeof page === 'string') {
             return Number(page);
         }
+        if (page == null) {
```

Comment:
should this be a === just because?

also, yay

Reply:
want to catch either undefined or null